File: python/search-by-index-id.py
import json
import requests

esconn = json.load(open('../esconn.json'))


r = requests.get("%s%s%s" % (esconn['uri'],'/node/profile/',278))
print(r.status_code, r.reason)
parsed = json.loads(r.content)
print (json.dumps(parsed, indent=4, sort_keys=True))

File: node/search-by-index-id.js
const util = require('util');
const request = require("request-promise");

let esconn = require("../esconn.json");


// search by index id
request.get({
    uri                    : esconn.uri + '/node/profile/278?pretty',
    resolveWithFullResponse: true,
    json                   : true
}).then((response) => {
    console.log('search by index id');
    console.log(`elasticsearch response : ${util.inspect(response.body, false, null)}`);
    return true
});






